app/src/main/res/values/strings.xml

```diff
@@ -683,6 +683,7 @@
     <string name="description_ends_with_punctuation">The text must not end with punctuation.</string>
     <string name="description_starts_with_article">Avoid starting with articles like “a” or “the”.</string>
     <string name="description_starts_with_uppercase">Start with a lowercase letter unless the first word is a proper noun.</string>
+    <string name="description_is_in_different_language">Your keyboard language and editing language are different. Are you sure you are editing in the correct language?</string>
```

Comment:
How about: `The text appears to be in the wrong language. The language should be %s.`

Reply:
"wrong" or "different" @dbrant

Reply:
How about "unexpected"?

Reply:
Perfect.. `The text appears to be in an unexpected language. The language should be %s.`